### ISSUE TITLE
🐛 [Bug Report] PathFinder.NormalizeZ = 1 makes npcs disappear below ground if there is terrain below

### ISSUE THREAD

**Issue by aleigood** [cmangos/issues#3226]
### Bug Details

![image](https://user-images.githubusercontent.com/3367999/203802447-b9f8e6b1-0224-4b27-bf34-b8ba1653c9c6.png)

### Steps to Reproduce

1.  .go xyz 7011.1 -6252.92 4.612382 530 
2.
3.
4.

### Core SHA1 Commit Hash

01b98b1fa68016b7c68de3c998aebbfe1d154940

### Database SHA1 Commit Hash

3efdb6e8fdec8d0e443907ddabdb3a0205ec13e6

### Operating System

Ubuntu

### Client Version

2.4.3 (The Burning Crusade)

**Comment by AnonXS** [comment]
![grafik](https://user-images.githubusercontent.com/19734826/203813833-88d3572c-6374-41cc-ae76-36f0eb731047.png)

5303271?

**Comment by aleigood** [comment]
> 5303271?
Yes, two server, but same problem.
![image](https://user-images.githubusercontent.com/3367999/203818116-ca1783f4-6a4f-42b7-a6d5-2993d084acb6.png)

**Comment by AnonXS** [comment]
well printout xyz from database for 5303271? because this is not where he is. (7014.39	-6252.89	46.5658)
looks like z value was divided by 10 on your server for mob, try fresh install.

**Comment by aleigood** [comment]
The xyz value is correct 7014.39 |   -6252.89 |    46.5658 , but I set the option PathFinder.NormalizeZ = 1, cause by this?

**Comment by AnonXS** [comment]
yes culprit is PathFinder.NormalizeZ = 1, db side is fine though, disable bad feature that is not doing what it should for Z axis and then in the wrong direction, normally you'd think it would "lift them a bit so they cant fall below z ground level, its doing the opposite for this npc because below the ground is a cave where it normalizes it to.

not sure why this even affects freshly spawning npcs, so disabling it for spawning npcs might fix the issue.

**Comment by AnonXS** [comment]
It's not really pathfinding/detour its some whatever we're doing internally with PathFinder.NormalizeZ = 1.
